L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -342,6 +346,21 @@ void l1t::GlobalBoard::receiveCaloObjectData(const edm::Event& iEvent,
       }  //end loop over Bx
     }
   }
+  if (receiveCICADA) {
+    edm::Handle<BXVector<float>> cicadaScoreHandle;
+    iEvent.getByToken(CICADAInputToken, cicadaScoreHandle);
+    if (not cicadaScoreHandle.isValid()) {
+      if (m_verbosity) {
+        edm::LogWarning("L1Tglobal") << "\nWarning: Input tag for the CICADA score"
+                                     << "\nrequested in configuration, but not found in the event.\n"
+                                     << "\nSetting score to 0.0";
+      }
+      setCICADAScore(0.0);
+    } else
+      setCICADAScore(cicadaScoreHandle->at(
+          0,
+          0));  //CICADA emulation will only provide a central BX, and one value. Unpacking may have more values, but that can't be guaranteed.
```

Comment:
Is it guaranteed that this will never crash ? In other words, should a check like
```cpp
if (not cicadaScoreHandle->isEmpty(0))
```
be added ?

Reply:
The only reason this would occur is there is a menu with CICADA in it, but there is no data in the uGT card, which is unlikely in a production scenario.

That being said, I did chase my tail for about an hour last night because I forgot this fact while trying to validate the unpacker collections before I realized there was no production unpacker stuff to validate.

I think it's better safe than sorry, so I can/will make this addition.

Reply:
Should be a guard for this in the most recent commit